app/models/rent.rb

```diff
@@ -0,0 +1,9 @@
+class Rent < ApplicationRecord
+  belongs_to :user
+  belongs_to :book
+
+  validates :from, presence: true
+  validates :to, presence: true
+  validates :user, presence: true
+  validates :book, presence: true
```

Comment:
You can put all the `validates` in one line
`validates :from, :to, :user, :book, presence: true`